.github/workflows/dockerCI.yml

```diff
@@ -60,13 +60,15 @@ jobs:
           VERSION=$(echo $VERSION | sed -e 's/^v//')
 
           # tag the image with :beta, :Version and :stable
-          docker tag image $IMAGE_ID:beta 
+          docker tag image $IMAGE_ID:dev 
```

Comment:
this is using the dev tag instead of beta like the other repos - is that a change?